paprika/build.py

```diff
@@ -5,15 +5,44 @@
 import logging as log
 
 
+def read_tleaplines(tleapfile, pdbfile=None, skip_solvate=True):
```

Comment:
So the goal is to have a template `tleap` input file packaged here?

Reply:
I think this code already existed inside the solvate function, but I pulled it out and gave it its own function.  Purpose: the user has to give us a tleap file that will build the system, so this function will read it and store the contents for manipulation/writing later.

Reply:
Hmm, okay.